styleguide/_themes/global/scss/fonts.scss

```diff
@@ -20,7 +21,7 @@ $code: Tahoma, Arial, Geneva, Helvetica;
 }
 
 @mixin copy($color: $gray-darkest) {
-  color: $gray-darkest;
+  color: $color;
```

Comment:
So after thinking on this more, this was actually behavior we had added in to the `heading()` and `subhead()` mixins for other reasons, and I think just for consistency I added it to `copy()` here.  We're not actually setting the color anywhere, but it's intact as a just-in-case method, but we could just scrap it if you'd prefer.